config.m4

```diff
@@ -537,6 +543,7 @@ if test "$PHP_MONGODB" != "no"; then
 mongodb was configured with the following options:
 
 Build configuration:
+  CC                                               : $CC
```

Comment:
This is helpful for debugging, as we'll see if `-std=gnu99` (or something else) was added to the compiler command.